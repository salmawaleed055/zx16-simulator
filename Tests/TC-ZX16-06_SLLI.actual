Loaded 10 bytes into memory
0x0000: 0271    xori x1, 1
0x0002: 2859    slli x1, 4
0x0004: 0271    xori x1, 1
0x0006: 2859    slli x1, 4
0x0008: FFC7    Unknown opcode

--- Final State ---
x0: 0x0000
x1: 0x0110
x2: 0x0000
x3: 0x0000
x4: 0x0000
x5: 0x0000
a0: 0x0000
a1: 0x0000
PC: 0x0008
---------------------

Simulation finished.
