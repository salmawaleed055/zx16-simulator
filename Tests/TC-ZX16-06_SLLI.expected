0x0000: 0271    li x1, 1
0x0002: 2859    slli x1, 4
0x0004: FFC7    ecall 0x3FF

--- Final State ---
x0: 0x0000
x1: 0x0010
x2: 0x0000
x3: 0x0000
x4: 0x0000
x5: 0x0000
a0: 0x0000
a1: 0x0000
PC: 0x0006
---------------------
Simulation finished.